src/utils/string.ts

```
@@ -3,6 +3,216 @@ const ansiRegex = [
+    // Check for emoji sequences
+    if (isEmoji(code)) {
+      width += 2;
+      i += charLen;
+
+      // Consume the rest of the emoji sequence:
+      // - Skin tone modifiers (Fitzpatrick scale)
+      // - Variation selectors
+      // - ZWJ + following emoji (family sequences like 👨‍👩‍👧‍👦)
+      while (i < stripped.length) {
+        const nextCode = stripped.codePointAt(i)!;
+        const nextCharLen = nextCode > 0xffff ? 2 : 1;
```

ESLint

[error] Invalid group length in numeric value.

(unicorn/numeric-separators-style)